# Without this patch I get the following abort when starting:
# > audacity
# 2023-07-01 17:50:20.337 Audacity[38837:d0b] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Error (1000) creating CGSWindow on line 263'
# *** First throw call stack:
# (
#         0   CoreFoundation                      0x00007fff9547d25c __exceptionPreprocess + 172
#         1   libobjc.A.dylib                     0x00007fff92903e75 objc_exception_throw + 43
#         2   CoreFoundation                      0x00007fff9547d10c +[NSException raise:format:] + 204
#         3   AppKit                              0x00007fff947f6e95 _NSCreateWindowWithOpaqueShape2 + 1403
#         4   AppKit                              0x00007fff947f5a21 -[NSWindow _commonAwake] + 3720
#         5   AppKit                              0x00007fff946d1400 -[NSWindow _commonInitFrame:styleMask:backing:defer:] + 882
#         6   AppKit                              0x00007fff946d0882 -[NSWindow _initContent:styleMask:backing:defer:contentView:] + 1054
#         7   AppKit                              0x00007fff946d0458 -[NSWindow initWithContentRect:styleMask:backing:defer:] + 45
#         8   AppKit                              0x00007fff94e2288d -[NSWindow init] + 95
#         9   libwx_osx_cocoau_core-3.1.dylib     0x000000010d0f8010 _Z33wxMacCocoaCreateCursorFromCGImageP7CGImageff + 79
#         10  libwx_osx_cocoau_core-3.1.dylib     0x000000010d0dd053 _ZN8wxCursor15CreateFromImageERK7wxImage + 287
#         11  libwx_osx_cocoau_core-3.1.dylib     0x000000010d0dcf1b _ZN8wxCursorC2ERK7wxImage + 33
#         12  Audacity                            0x000000010af33f88 _Z10MakeCursoriPKPKcii + 264
#         13  Audacity                            0x000000010ad7dbd6 _GLOBAL__sub_I_AdornedRulerPanel.cpp + 2678
#         14  ???                                 0x00007fff6da94c6e 0x0 + 140735033199726
#         15  ???                                 0x00007fff6da94dfa 0x0 + 140735033200122
#         16  ???                                 0x00007fff6da91aa2 0x0 + 140735033186978
#         17  ???                                 0x00007fff6da91936 0x0 + 140735033186614
#         18  ???                                 0x00007fff6da851da 0x0 + 140735033135578
#         19  ???                                 0x00007fff6da884ca 0x0 + 140735033148618
#         20  ???                                 0x00007fff6da8427b 0x0 + 140735033131643
#         21  ???                                 0x00007fff6da8405e 0x0 + 140735033131102
# )
# libc++abi.dylib: terminating with uncaught exception of type NSException
# Abort
# 

diff --git a/src/tracks/ui/SelectHandle.cpp b/src/tracks/ui/SelectHandle.cpp
index 41cc84640d021ba93280e7660b32e3386b93ee14..1ea19922908dc12438ce585db4529e330d849b9a 100644
--- a/src/tracks/ui/SelectHandle.cpp
+++ b/src/tracks/ui/SelectHandle.cpp
@@ -297,8 +297,14 @@ namespace
       (SelectionBoundary boundary, bool frequencySnapping,
        TranslatableString &tip, wxCursor *&pCursor)
    {
+#ifdef __WXMAC__
+      // don't use those pesky and unclear pointing-finger cursors!
+      static wxCursor adjustLeftSelectionCursor{ wxCURSOR_CROSS };
+      static wxCursor adjustRightSelectionCursor{ wxCURSOR_CROSS };
+#else
       static wxCursor adjustLeftSelectionCursor{ wxCURSOR_POINT_LEFT };
       static wxCursor adjustRightSelectionCursor{ wxCURSOR_POINT_RIGHT };
+#endif
 
       static auto bottomFrequencyCursor =
          ::MakeCursor(wxCURSOR_ARROW, BottomFrequencyCursorXpm, 16, 16);
